python/python/bystro/supervised_ppca/gf_generative_pt.py

```
@@ -606,6 +606,71 @@
                 nn.Softplus()(trainable_variables[1]).detach().numpy()
             )
 
+    def transform(self, X: NDArray[np.float_], sherman_woodbury: bool = False):
```

Ruff

`np.float_` will be removed in NumPy 2.0. Use `numpy.float64` instead.

Replace with `numpy.float64`

(NPY201)